.github/workflows/docker-image-build-push.yml

```diff
@@ -0,0 +1,30 @@
+# Build and push docker image
+name: Docker Image CI
+
+on:
+  push:
+    branches:
+      - production
+      - master
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: |
+            ${{ secrets.DOCKER_USERNAME }}/atd-executive-dashboard:production
+            ${{ secrets.DOCKER_USERNAME }}/atd-executive-dashboard:latest
+      - name: Image digest
+        run: echo ${{ steps.docker_build.outputs.digest }}
```

Comment:
I think all I need to do to configure this is to add the secrets? Github will just take it from here?